### PR TITLE
scripts/concourse.check: add support for teams

### PR DESCRIPTION
**Pull request by Amit-PivotalLabs**

### CODE REVIEW

File: scripts/concourse.check [hunk 1]
```diff
@@ -80,7 +85,7 @@ class Concourse
   private
 
   def fetch_latest_build
-    job_url = "#{@server_url}/api/v1/pipelines/#{@pipeline_name}/jobs/#{@job_name}"
+    job_url = "#{@server_url}/api#{@team ? "/team/" + @team : ""}/pipelines/#{@pipeline_name}/jobs/#{@job_name}"
```
Comment (vito) [inline scripts/concourse.check]: This should be `/api/v1/teams/foo/[...]`

**Comment by Amit-PivotalLabs** [comment]
Thanks @vito does it look right now?

**Comment by jbheron** [comment]
The CF docs team benefits from this PR. Thanks.

**Comment by vito** [comment]
Aside from https://github.com/concourse/fly/pull/103#discussion_r77756694 yeah (I think it's weird for the team to come last).

**Comment by pythe** [comment]
The Tracker team would also benefit from this. The build failure looks like we're getting HTML and a redirect where we expect JSON.

**Comment by mkb** [comment]
👍 

We could use this at Stark & Wayne as well.

**Comment by cppforlife** [comment]
will release version today.

**Comment by antitux** [comment]
+1 on this, for CSAA DI

**Comment by oppegard** [comment]
Bump 😁

**Comment by Amit-PivotalLabs** [comment]
Hey @cppforlife, you mentioned you'd be able to cut a release a couple days ago.  Sounds like a large number of people are waiting on this.  Anything I can do to expedite this?  I've got a few hundred builds to watch and Checkman is the only way I can do this effectively (thanks!).

**Pull request merged**